python/paddle/fluid/framework.py

```diff
@@ -948,7 +884,7 @@ def get_new_list_tensor(old_list):
             attrs=attrs)
 
         out = slice_out_var
-    elif use_strided_slice == True and len(slice_axis) > 0:
+    elif use_strided_slice == True and len(axes) > 0:
```

Comment:
Can we replace `elif use_strided_slice == True and len(axes) > 0` with `elif use_strided_slice and len(axes) > 0` ?

Usually if a variable `x` is boolean, we write `if x` instead of `if x is True` or `if x == True`

Reference from official programming recommendations:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

![image](https://user-images.githubusercontent.com/7913861/118075348-6bf2ff00-b3e2-11eb-92af-0d77c2f46321.png)

Reply:
Thanks, done.